textproto/header.go

```diff
@@ -513,17 +513,17 @@ func foldLine(v string, maxlen int) (line, next string, ok bool) {
 		}
 	} else {
 		// Find the last QP character before limit
-		foldAtQP := qpReg.FindAllStringIndex(v[:foldBefore], -1)
+		foldAtWSP := wspReg.FindAllStringIndex(v[:foldBefore], -1)
```

Comment:
Can we not use `regexp` here? It'd be easy to replace with `unicode.IsSpace` and functions from `strings`.

Reply:
Well, that's definitely better not to have a regex here. Now I've effectively reverted #22. 
However now I have a slight concern about `\n` handling. If theoretically input contains `\n` without corresponding `\r` we will have an incorrect output. I'm not sure about practical possibilities of this though. Any encoder should take care of this `\n` *beforehand*